lib/cosmos/config/config_parser.rb

```diff
@@ -158,7 +171,10 @@ def parse_file(filename,
                    remove_quotes = true,
                    &block)
       @filename = filename
-      file = File.new(@filename, 'r')
+      # Create a temp file where we can write the ERB parsed output
+      file = Tempfile.new("parsed_#{File.basename(filename)}")
+      file.write(ERB.new(File.read(@filename)).result(binding))
+      file.rewind
```

Comment:
Right now target.rb only grabs *.txt files in the cmd_tlm directory. Not sure why we did that but obviously we'd have to change that as well. My thought was that we'd just parse our existing text files and if there are things to interpret we do so. I ran this against a complex server configuration and it doesn't seem to really impact the load time. Specifying the extension would allow for future template engines but I'm not sure that really applies to us the same way it does for Rails.